nbs/01_Tutorials/01_tutorial.ipynb

```diff
@@ -16,7 +16,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe write more about the order, perhaps say sort your notebooks according to a flow of some kind flow or narrative </p><p> </p><p>it’s not necessarily the order you created it in but the order you want it to be?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>